Source/WebInspectorUI/UserInterface/Views/ResourceClusterContentView.js

```diff
@@ -25,13 +25,14 @@
 
 WI.ResourceClusterContentView = class ResourceClusterContentView extends WI.ClusterContentView
 {
-    constructor(resource)
+    constructor(resource, {inNetworkView} = {})
```

Comment:
i think we should match the parameter name used by `WI.ImageResourceContentView` (i.e. `disableDropZone`) to allow for future engineers searching for `disableDropZone` to easily see the entire codepath in the search results instead of having to do a second search